### PR TITLE
TLF-100 - COA - Privacy policy Page

### PR DESCRIPTION
**Pull request by vinodhasamiyappanHO**
### What

TLF-100 - Added Privacy policy page [TLF-100](https://collaboration.homeoffice.gov.uk/jira/browse/TLF-100)

### Why

Adding privacy policy page newly to the COA flow.

### How

- Added the privacy policy page and request submitted(confirmation) page after check answers screen.

### Testing
- [x] Local testing passed
- [x] Test Lint Passed in Local
- [] Pipeline Check Passed
- [] Branch Testing passed

### CODE REVIEW

File: apps/coa/translations/src/en/buttons.json [hunk 1]
```diff
@@ -11,5 +11,6 @@
   "proof-of-identity": "Proof of identity",
   "what-you-need": "What you will need",
   "proof-of-address": "Proof of Address",
-  "start-now": "Start now"
+  "start-now": "Start now",
+  "submit-report": "Confirm and submit"
```
Comment (dk4g) [inline apps/coa/translations/src/en/buttons.json:15]: The button wording should be "Submit change"
I would also suggest to rename `submit-report` key, as could be misleading. Perhaps just `submit` or `submit-change`

File: apps/coa/index.js [hunk 2]
```diff
@@ -196,7 +196,19 @@ module.exports = {
...
       behaviours: [Summary, ModifyChangeURL],
       sections: require('./sections/summary-data-sections'),
-      template: 'summary'
+      template: 'summary',
+      next: '/privacy-policy'
+    },
+    '/privacy-policy': {
+      behaviours: [Summary],
+      fields: ['privacy-check'],
+      sections: require('./sections/summary-data-sections'),
```
Comment (dk4g) [inline apps/coa/index.js:205]: no need for `behaviours` and `sections` on this step

File: assets/js/index.js [hunk 3]
```diff
@@ -11,3 +11,12 @@ $('.typeahead').each(function applyTypeahead() {
...
 });
+
+$(function () {
+  if ($('#loader-container')) {
+    $('.govuk-button').on('click', function () {
+      $('#loader-container').addClass('spinner-loader');
+      $('#report-submit').addClass('visuallyhidden');
+    });
+  }
+});
```
Comment (dk4g) [inline assets/js/index.js:22]: Can we avoid using jQuery, so we can deprecate it from this project?

**Review by dk4g — CHANGES_REQUESTED** [review]

File: apps/coa/translations/src/en/buttons.json (hunk 1 above)
Comment (vinodhasamiyappanHO) [reply apps/coa/translations/src/en/buttons.json]: Updated the button key as "submit" and label should be "Confirm and submit" to matches the figma.

File: apps/coa/index.js (hunk 2 above)
Comment (vinodhasamiyappanHO) [reply apps/coa/index.js]: Removed this from the step for privacy-policy

File: assets/js/index.js (hunk 3 above)
Comment (vinodhasamiyappanHO) [reply assets/js/index.js]: removed jQuery functions and made changes with equivalent javascript functions.

**Review by MikeMaratheftis — APPROVED** [review]

**Review by RobertMcCann — APPROVED** [review]
looks good to me, remember to squash/summarise your commits when you merge :)

**Pull request merged**